src/components/StationSlider/StationSlider.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ScrollView, View } from 'react-native';
+import { deviceProps } from 'helpers/device';
+import {
+  AttributionView, BottomSectionView, CalloutSectionView,
+  CreditSectionView, MapboxIconImage
+} from './StationSliderCss';
+import StationCard from '../StationCard';
+import AttributionButton from '../AttributionButton';
+import { blueStops } from 'helpers/config';
+import { mapboxIcon } from 'assets/icons/mapbox-icon';
+
+export default class StationSlider extends React.Component {
+
+  static propTypes = {
+    activeStationIndex: PropTypes.number,
+    connected: PropTypes.bool.isRequired,
+    loading: PropTypes.bool.isRequired,
+    mode: PropTypes.string.isRequired,
+    nearestStationIndex: PropTypes.number,
+    showCallout: PropTypes.func.isRequired,
+    stationDistances: PropTypes.array,
+    navigation: PropTypes.object.isRequired
+  }
+
+  state = {
+    stationIndex: 0,
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { activeStationIndex } = this.props;
+    const scrollView = this.stationScrollView;
+
+    const newStationIndexProp = (activeStationIndex !== nextProps.activeStationIndex);
+    const newStationIndexState = (nextProps.activeStationIndex !== this.state.stationIndex);
+
+    if (newStationIndexProp && newStationIndexState) {
+      // console.group('Indexes')
+      // console.debug('stationIndex: ', this.state.stationIndex)
+      // console.debug('activeStationIndex: ', nextProps.activeStationIndex)
+      // console.groupEnd()
+
+      // console.group(newStationIndexProp && newStationIndexState)
+      // console.log('newStationIndexProp: ', newStationIndexProp)
+      // console.log('newStationIndexState: ', newStationIndexState)
+      // console.groupEnd()
+
+      scrollView.scrollTo({
+        x: nextProps.activeStationIndex * deviceProps.deviceScreen.width,
```

Comment:
Is it possible to tell ScrollView to scroll past the end? User might be confused if that is possible and it happens